### PR TITLE
Update rumble.json

### PR DESCRIPTION
**Pull request by Jordiyy**
FN Buggy, Raid Judge

### CODE REVIEW

File: common/data/rumble.json [hunk 1]
```diff
@@ -53987,6 +53987,431 @@
...
     },
+    {
+      "ability": [
+        {
+          "effects": [
+            {
+              "attributes": [ "ATK" ],
+              "effect": "buff",
+              "level": 1,
+              "targeting": { "targets": [ "INT" ] }
```
Comment (ThePieBandit) [inline common/data/rumble.json:53998]: the current pattern for types only is to wrap them in [], so "[INT]". It's a holdover from the way things used to be done. There's some exception handling so it won't break things, but it's good to be consistent. Again, this is for types only.

File: common/data/rumble.json [hunk 2]
```diff
@@ -53987,6 +53987,431 @@
...
+      "special": [
+        {
+          "cooldown"; 18,
+          "effects": [
+            {
+              "attributes": [ "Blow Away" ],
+              "duration": 10,
+              "effect": "buff",
+              "level": 1,
+              "targeting": { "targets": [ "INT" ] }
```
Comment (ThePieBandit) [inline common/data/rumble.json:54060]: same, "[INT]"

File: common/data/rumble.json [hunk 3]
```diff
@@ -53987,6 +53987,431 @@
...
+            },
+            {
+              "override":  { "level": 3, "duration": 19 }
+            }
+          ]
+        }
+      ],
+      "stats": {
+        "def": 215,
+	      "rumbleType": "SPT",
```
Comment (ThePieBandit) [inline common/data/rumble.json:54407]: fix indentation please

**Review by ThePieBandit — COMMENTED** [review]
A few tiny comments

File: common/data/rumble.json (hunk 3 above)
Comment (Jordiyy) [reply common/data/rumble.json:54407]: indentation on my atom view is correct but once on github shows that way. I'll correct those things and add new characters in my next pull

**Comment by Jordiyy** [comment]
The new pull there're more characters and fixed those things

**Pull request closed without merging**